ml-agents/mlagents/trainers/components/reward_signals/reward_signal.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from mlagents.trainers.trainer import UnityTrainerException
+from mlagents.trainers.policy import Policy
+import numpy as np
+import abc
+
+import tensorflow as tf
+
+logger = logging.getLogger("mlagents.trainers")
+
+
+class RewardSignal(abc.ABC):
+    value_name = "Policy/{} Value Estimate".format(__name__)
```

Comment:
Not sure `__name__` is what you want here. I think it will evaluate to something like
`mlagents.trainers.components.reward_signals.reward_signal`